encoder.go

```diff
@@ -62,7 +129,7 @@ func NewEncoder(sample_rate int, channels int, application Application) (*Encode
 // Init initializes a pre-allocated opus encoder. Unless the encoder has been
 // created using NewEncoder, this method must be called exactly once in the
 // life-time of this object, before calling any other methods.
-func (enc *Encoder) Init(sample_rate int, channels int, application Application) error {
+func (enc *Encoder) Init(sample_rate int, channels int, application int) error {
```

Comment:
here, too, I'm not a fan of including these unrelated changes in this patch. this patch is about adding encoder settings, but these changes are not necessary for that. I'm not opposed to them (I agree; they should be const, not var), but this PR is not the place to do that.